src/odemis/gui/cont/streams.py

```diff
@@ -78,13 +78,15 @@ class StreamController(object):
     """ Manage a stream and its accompanying stream panel """
 
     def __init__(self, stream_bar, stream, tab_data_model, show_panel=True, view=None,
-                 sb_ctrl=None):
+                 sb_ctrl=None, hw_settings_display_order=None):
         """
         view (MicroscopeView or None): Link stream to a view. If view is None, the stream
         will be linked to the focused view. Passing a view to the controller ensures 
         that the visibility button functions correctly when multiple views are present.
         sb_ctrl (StreamBarController or None): the StreamBarController which (typically)
           created this StreamController. Only needed for ROA repetition display.
+        hw_settings_display_order (None or list): A custom display order for the hardware settings.
```

Comment:
By following the none placements from the other input arguments.
```suggestion
        hw_settings_display_order (list or None): A custom display order for the hardware settings.
```